### ISSUE TITLE
JSNM - Data Entry

### ISSUE THREAD

**Issue by cefilipek** [ArctosDB/data-migration#189]
HI! I was wondering if there is a way to add the object location into the record, but not make it visible to the public? It looked like there was on the tutorials and in other institution's records, but I was unable to find a space for it on the data entry form.

Thanks!

**Comment by Jegelewicz** [comment]
The best way to add an object location is with the use of object tracking and barcodes. See [How to Start Object Tracking in Arctos](http://handbook.arctosdb.org/how_to/How-to-Start-Object-Tracking.html). Object tracking information done this way is not available on public pages.

The second best option is to add the locality attribute to the part. You can see an example of this here: http://arctos.database.museum/guid/ALMNH:ES:1

As far as I know, this information cannot be encumbered at this time. @dustymc is that the case? If so, how hard would that be to change?

Also, there currently isn't any good documentation for adding the part attributes, but if this is the route you want to go, I can help you and we can write some documentation together.

**Comment by cefilipek** [comment]
We use barcodes, but just for the objects themselves, so it seems like adding a part attribute would be best if we were able to encumber the information. I also don't have the object tracking tab in Arctos. I did find a locality attribute that was masked, but I don't know if that means it's simply not there or was made to be invisible to the public. I will link that entry here: http://arctos.database.museum/guid/CHAS:Egg:10

**Comment by Jegelewicz** [comment]
Yes! that attribute can be encumbered! This will be a good option for you. The location attributes can be added during data entry (if data are entered manually) or as a seperate upload after the original records have been loaded using the [Bulkload Specimen Part Attribute tool](http://arctos.database.museum/tools/BulkloadSpecimenPartAttribute.cfm).

**Comment by cefilipek** [comment]
Great! As of now most of our objects will be entered manually. How do I add the location attribute to the data entry form? I did find the encumbrance page. Would the information be hidden on all objects I add if I put in an encumbrance for "mask part attribute location"?

Thanks!

**Comment by Jegelewicz** [comment]
> I did find the encumbrance page; would the information be hidden on all objects I add if I put in an encumbrance for "mask part attribute location"?

If all of the catalog records are added to the encumbrance, then all part location attributes will be hidden.

> How do I add the location attribute to the data entry form?

When entering a record don't add a part. Instead, use the "Add more" at the bottom of the data entry screen and select "add specimen part". Documentation is not written, but here is the thread discussing how it works: https://github.com/ArctosDB/documentation-wiki/issues/132

http://handbook.arctosdb.org/how_to/How-to-load-Data-Entry-linked-Attributes.html

**Comment by cefilipek** [comment]
Thank you!

**Comment by cefilipek** [comment]
I'm having another problem with this section of the form. I used the Add Specimen Part extension (maybe not the right word) so that I could add the object location and encumber it, but it didn't save even though I selected "Save Part". I went in to edit it, but it appears I do not have access to that page nor do I have access to media, which I will need at some point since we have images we want to attach. Is there a way I can get access to these features?
![Error Message](https://user-images.githubusercontent.com/56844074/72828781-fd18f200-3c42-11ea-9983-c119ce208802.png)
![Media Authorization](https://user-images.githubusercontent.com/56844074/72828794-05712d00-3c43-11ea-87bd-cf747ef8a5ad.png)
